app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -50,6 +51,17 @@ open class FirefoxApplication : LocaleAwareApplication() {
             // Enable crash reporting. Don't add anything above here because if it crashes, we won't know.
             SentryIntegration.init(this, serviceLocator.settingsRepo)
 
+            // Set up Glean by connecting it to the preference flag for data collection and calling
+            // the initialize() function.
+            with(serviceLocator.settingsRepo) {
```

Comment:
Nit: Since we only have one receiver here, I think it's a bit simpler to avoid using the `with` syntax here.